add panel to say Login Policy is not on Free Tier (#259)
spacelift-io/user-documentation: docs/concepts/policy/login-policy.md

Login policies can allow users to log in to the account, and optionally give them admin privileges, too. Unlike all other policy types, login policies are global and can't be attached to individual stacks. They take effect immediately once they're created and affect all future login attempts.

!!! info

>     Login Policies are only evaluated when using the Cloud or Enterprise Tier.

Reviewer: ```suggestion
    Login policies are only evaluated for the [Cloud or Enterprise plan](https://spacelift.io/pricing).
```
